### ISSUE TITLE
wherever there is <hi rend="i">, its value needs to be rendered in italics and displayed

### ISSUE THREAD

**Issue by foxm** [blakearchive/archive#295]
i see this info missing from copy information and electronic edition information

**Comment by foxm** [comment]
example: 

`<note>Object 1 as arranged and numbered in this copy, the frontispiece to <hi rend="i">Songs of Innocence</hi>, precedes Object 2, the general title page; Object 29 as arranged and numbered in this copy, the frontispiece to <hi rend="i">Songs of Experience</hi>, faces Object 30, the title page to <hi rend="i">Songs of Experience</hi>.</note>`

**Comment by queryluke** [comment]
from #282 

> it's missing text within tags within the illustration description tags.
for example,
<hi rend="i">A Midsummer Night's Dream</hi>
is not showing in the illustration description for http://localhost:8002/blake/copy/s-los.b?descId=s-los.b.illbk.05

**Comment by queryluke** [comment]
Looks like this is 2 pronged.

# Electronic Info
This just needed css entry in the tei.css file. The Electronic Info should now be properly itlaicized.

# Copy Information & Illustration Description
@nathan-rice it looks like <hi> and all content between the <hi> tag element are removed. Thoughts? Could we run a regex to replace <hi rend="i"> with <em>?

**Comment by nathan-rice** [comment]
This was the result of a bug in the xslt.  I've updated them to work correctly in this case.

**Comment by foxm** [comment]
hmm, still seeing the issue. the content of the \<hi\> tags is removed in the illustration description and copy info

**Comment by nathan-rice** [comment]
Did you re-populate the database?

**Comment by foxm** [comment]
yeah, re-populated, and checked the two examples given in this issue

**Comment by nathan-rice** [comment]
The issue here is that you have "presentational" xml inside xml data structures.  The conversion from XML to JSON is basically lossless for well formed XML, but misused XML (such as tags interleaved in text like this) is one of the area where loss does occur.  There are two ways forward:

1. Slurp in the illusdesc tag (and any other tags where you want presentation) as text, and run an XSLT on them.  This gives you italic titles, but you are going to lose any structure to the data, it is going to come across as just a plain string.  This seems like a reasonable path forward for the illustration description, as there doesn't seem to be metadata there for which the xml structure needs to be preserved.
2. Strip out the presentation tags, and live without the italics.  This is probably going to be what needs to happen for copy information.

**Comment by foxm** [comment]
1 sounds fine, but we absolutely need the italics in both places, so we'll have to come up with a different solution for Copy Information

**Comment by queryluke** [comment]
We had an epiphany we should use markdown in the bad styling. I'll test this and report back

**Comment by queryluke** [comment]
@foxm @joeafletch 

Using markdowns in the bads will work. So you'll need to change all the `<hi rend="i">string</hi>` to `*string*`. HOWEVER, you should only do this in the copy header/source blocks and the illustration descriptions. 

To clarify why: copy header/source and Illustration descriptions get parsed out into JSON. Then we rebuild the information.

But the electronic information and textual transcriptions are still getting run through the XSLT, which needs the `<hi rend="i">`

**Comment by foxm** [comment]
ok, i've tasked this out to adam e.

**Comment by foxm** [comment]
@queryluke i noticed that 'A Midsummer Night's Dream' in the illustration description for http://london.lib.unc.edu/blake/copy/s-los.b?descId=s-los.b.illbk.05 is italicized, which is how it's supposed to be. the BAD is showing the <hi> tags around the string. it seems to be working with the tags. do you want these changed to asterisks?

**Comment by foxm** [comment]
@queryluke it looks like it's the notes in transcriptions that aren't rendered in italics at the moment:

see editors' notes here: http://london.lib.unc.edu/blake/copy/america.a?descId=america.a.illbk.18

'America' should be italicized

**Comment by queryluke** [comment]
I reviewed where this is being generated, and it should have the markdown syntax applied.

The only 2 parts of the BADs or Info files that should be avoided are the Textual Transcription and Electronic Information. Both of those get run through XSLT, which generates an HTML output.

**Comment by foxm** [comment]
see my last two comments for references. `<note>` tags under transcriptions (`<phystext>`) are not being rendered in italics:

`<lg type="prose" justify="left">
<l n="america.a.illbk.18.26" justify="left" indent="27">
FINIS
<note>
The word "FINIS" was etched in white line in the tail of the serpent at the bottom of the plate. In some other copies of
<hi rend="i">America</hi>
, such as Copies O and M, the word has been obscured by color.
</note>
</l>
</lg>`

and illustration descriptions (`<illusdesc>`) have `<hi>` tags in them that are being rendered as italics:

`<illusobjdesc>
A male figure, wearing a gown and a separate cloak, sits on a lily. If the flower determines the scale, then this diminutive male may be a fairy. He is crowned and holds a scepter in his left hand; he may be a king. He props up his upper body with his right arm; his right leg is drawn up and his left leg dangles from the edge of the flower. He faces forward and has long hair and a beard. He has been identified by modern critics as either Har (see Plate 4) or Oberon from Shakespeare's
<hi rend="i">A Midsummer Night's Dream</hi>
.
</illusobjdesc>`

**Comment by queryluke** [comment]
The confusion where having is that I'm looking at the resulting Object Model on the client side of the application. I've always been a little fuzzy about how things get translated from the BADs to the Object Model. Anyway, here is my attempt to clear things up:

1. Go to: http://london.lib.unc.edu/blake/copy/america.a?descId=america.a.illbk.18
2. Open the developer tools (CMD+OPT+i)
3. In the **Console** tab, you'll see print outs of the client side models
4. At the bottom, you should see:
```
getting supplemental objects
getting objects with textual references
object
>Object
```
5. Expand Object (here you can look though all the attributes)
6. Any property that begins with `<div xmlns:exist="http://exist.sourceforge.net/NS/exist">` has been run through the XSLT
7. For the Object Model:

**markup_text** is the textual transcription.  Notice that:
  * what is a `<note>` in the BAD becomes `<td class="tei-line-note"/>` in the mark_up text.
  * `<note>` tags are not rendered at all. 
  * Any `<note>` tag in the BAD appears in the **notes** attribute of the Object Model
  * Meaning `<note>` tags are not run through XSLT. So for objects, the `<note>` tags should have markdown styling applied.

**illustration_description[#text]** is the illustration description. In addition, expand **components[0].component.illusobjectdesc**, and note the individual components are getting run through the XSLT as well.

The Copy Model has the attribute **header_html** which is the XSLT for the electronic information. So, again, leave the `<hi rend="i">` here.

So in summary, there are 3 sections of the BAD being run through the XSLT:
1. the textual transcription (but not `<notes>`)
2. any illustration descriptions, which appear to correspond with `<illusobjdesc>`
3. Copy header, which appears to correspond with everything in the `<header>`

Everything else should have markdown syntax applied.

Clear as mud now?

**Comment by foxm** [comment]
Almost. I see that `<note>` tags in Copy Information (`<objdesc>`) should have markdown, but what about the rest of Copy Information?

**Comment by foxm** [comment]
ok, as i see it now, you're saying these should receive markdown:

`<note>`s no matter where they are
Copy Information (`<objdesc>`)
work info files

**Comment by queryluke** [comment]
Yes, that's correct.

**Comment by foxm** [comment]
@queryluke  ok, i've uploaded new BADs and info files. they've got the italics in markdown in the right places. not sure if you've turned on markdown, but now i'm seeing the words with *'s around them in Copy Information and Editors Notes

**Comment by foxm** [comment]
@queryluke examples:

work info: http://london.lib.unc.edu/blake/work/mhh
editors' notes: http://london.lib.unc.edu/blake/copy/america.a?descId=america.a.illbk.18
copy information (*Our End is come*): http://london.lib.unc.edu/blake/copy/mhh.b?descId=mhh.b.illbk.01

**Comment by foxm** [comment]
@queryluke the editors' notes and copy information still need the markdown turned on

**Comment by queryluke** [comment]
I didn't deploy the code
